src/model/cve.js

```diff
@@ -110,14 +102,11 @@ CveSchema.statics.validateRejected = function (record) {
   return false
 }
 
-CveSchema.statics.updateCveToRejected = function (id, providerOrgId, record, newCnaContainer) {
+CveSchema.statics.updateCveToRejected = function (id, providerMetadata, record, newCnaContainer) {
   record.containers.cna = newCnaContainer.cnaContainer // replace cna field on existing record
   const updateDate = (new Date()).toISOString()
```

Comment:
Either use the timestamp in the providerMetadata, or replace that timestamp with this one so that they're identical